ga/developer/javaee8/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# (C) Copyright IBM Corporation 2018.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM websphere-liberty:kernel
+
+ARG REPOSITORIES_PROPERTIES=""
+
+RUN if [ ! -z $REPOSITORIES_PROPERTIES ]; then mkdir /opt/ibm/wlp/etc/ \
+  && echo $REPOSITORIES_PROPERTIES > /opt/ibm/wlp/etc/repositories.properties; fi \
+  && installUtility install --acceptLicense \
+    appSecurity-2.0 collectiveMember-1.0 ldapRegistry-3.0 \
+    localConnector-1.0 monitor-1.0 requestTiming-1.0 restConnector-2.0 \
+    sessionDatabase-1.0 ssl-1.0 transportSecurity-1.0 webCache-1.0 webProfile-8.0 \
```

Comment:
we should also add `sessionCache-1.0` here, which was a new feature in 18.0.0.1 similar to `sessionDatabase-1.0`